From 83774b612a39ae81d1a4c9a4ca14278cd9e674e7 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Thu, 21 Apr 2016 19:10:52 +0200
Subject: [PATCH 1/1] mod_cv: remove unneeded includes

OpenCV contains various modules which have inter-module dependencies.
While including freeswitch into buildroot we stumbled over the fact
that mod_cv.cpp includes more headers than needed[1]. These OpenCV
headers are generic headers which require more OpenCV modules enabled
than really used by Freeswitch. The attached fixes the problem.

[1] http://article.gmane.org/gmane.comp.lib.uclibc.buildroot/146662

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
[Patch sent upstream: https://freeswitch.org/jira/browse/FS-9093]
---
 src/mod/applications/mod_cv/mod_cv.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/mod/applications/mod_cv/mod_cv.cpp b/src/mod/applications/mod_cv/mod_cv.cpp
index 5183adc..5c10b8d 100644
--- a/src/mod/applications/mod_cv/mod_cv.cpp
+++ b/src/mod/applications/mod_cv/mod_cv.cpp
@@ -40,9 +40,6 @@ using namespace cv;
 
 #include <switch.h>
 
-#include <cv.h>
-#include "cvaux.h"
-#include "cxmisc.h"
 #include "highgui.h"
 #include <stdio.h>
 #include <stdlib.h>
-- 
2.8.0.rc3

